scripts/skills/actives/rf_blitzkrieg_skill.nut

```diff
@@ -33,7 +33,14 @@ this.rf_blitzkrieg_skill <- ::inherit("scripts/skills/skill", {
 			id = 10,
 			type = "text",
 			icon = "ui/icons/special.png",
-			text = ::Reforged.Mod.Tooltips.parseString("Every ally within " + ::MSU.Text.colorPositive("4") + " tiles who has at least " + ::MSU.Text.colorNegative("10") + " [Fatigue|Concept.Fatigue] remaining and is not [stunned|Skill+stunned_effect] or [fleeing|Concept.Morale] will get the [Adrenaline|Skill+adrenaline_effect] effect and build " + ::MSU.Text.colorNegative("10") + " [Fatigue|Concept.Fatigue]")
+			text = ::Reforged.Mod.Tooltips.parseString("You and allies of your faction within " + ::MSU.Text.colorPositive("4") + " tiles who have at least " + ::MSU.Text.colorNegative("10") + " [Fatigue|Concept.Fatigue] remaining gain the [Adrenaline|Skill+adrenaline_effect] effect and build " + ::MSU.Text.colorNegative("10") + " [Fatigue|Concept.Fatigue]")
+		});
+
+		ret.push({
+			id = 19,
+			type = "text",
+			icon = "ui/icons/warning.png",
+			text = "Will not affect allies who are [fleeing|Concept.Morale], [stunned|Skill+stunned_effect], or [sleeping|Skill+sleeping_effect]"
```

Comment:
Mentioning `sleeping` here is definitely an improvement. Ideally we should introduce a new `Stunned` concept which lists all the various effects which count as stunning a person i.e. the effects which set `properties.IsStunned` to `true`. I say this because there are other effects which do that e.g. the `horrified_effect`. But we don't have to do that immediately.

Reply:
I thought about this as I was doing these. I agree, sounds like an improvement. Perhaps a new concept called incapacitated instead of putting it all under stun.

Reply:
So for now keep it as you've done and once we have the #466 PR merged in we can then redo the relevant tooltips to use those concepts.